.gitignore

```diff
@@ -62,4 +62,4 @@ typings/
 
 # parcel
 .cache
-dist
+dist/*
```

Comment:
I think just adding dist will also work due to latest configurations. Because that was working in my case.